src/App.jsx

```diff
@@ -38,9 +38,12 @@ const Collections = ({ projects, projectKey, title, currentFilter }) => {
 const App = (props) => {
   const { title, projectKeys, projects, initialFilter } = props;
   const [currentFilter, applyFilter] = useState(initialFilter);
+  const defaultWhite = window.matchMedia(
+    "(prefers-color-scheme: white)"
```

Comment:
```suggestion
    "(prefers-color-scheme: light)"
```

See [specs](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme)

Right now, your code does not make it possible to use the light theme. However, after the proposal above, the dark mode is not working anymore:

![image](https://user-images.githubusercontent.com/3101914/149745503-8e380e44-ffc9-4d73-afb6-507ba585962e.png)

Additionally, the current code uses dark mode by default if the use has no preference at all, which I personally don't like as a default value. We should probably invert the logic.